src/src/userauth/userauth.service.ts

```diff
@@ -154,6 +156,36 @@ export class UserauthService {
 					);
 				}
 
+				let user_id = result?.data?.id;
+
+				if (user_id) {
+					// Set the timezone to Indian Standard Time (Asia/Kolkata)
+					const formattedISTTime =
+						await this.method.getFormattedISTTime();
+
+					// Format the time as per datetime
+
+					let acknowledgement_create_body = {
```

Comment:
@GitVivekHub create core service for this and call and use it from other services.